07-plotting-solar-data/displaying_data.ipynb

```diff
@@ -90,12 +125,22 @@
      "outputs": []
     },
     {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "But it can also be used for more complex plots:"
+     ]
+    },
+    {
      "cell_type": "code",
      "collapsed": false,
      "input": [
+      "plt.close()\n",
```

Comment:
So I can be sure it's definitely only plotting  what I'm telling it to do, and stuff from the previous figure isn't ending up there, though I'm sure it probably isn't anyway. I'll gladly cut all instances of this before we run it.